Dockerfile

```diff
@@ -18,10 +18,12 @@ COPY go.sum go.sum
 COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
+COPY hack/ hack/
 COPY vendor/ vendor/
+COPY version/ version/
```

Comment:
You need to copy `.git` as well, otherwise the git commands in build.sh won't work.
Remember to update the downstream Dockerfile as well!

Reply:
I totally forgot the downstream. I'll open an MR
Thanks

Reply:
I totally forgot the downstream. I'll open an MR

Thanks